Modules/Directory.Build.props

```diff
@@ -12,4 +12,21 @@
             CS8777,CS8819,CS8824,CS8825,CS8847
         </WarningsAsErrors>
     </PropertyGroup>
+
+    <ItemGroup Condition="'$(Configuration)' != 'Release' And !$(MSBuildProjectFile.Contains('Cleanup.csproj')) And !$(MSBuildProjectFile.Contains('SanityCheck.csproj')) And !$(MSBuildProjectFile.Contains('Devices.AdminCli.csproj'))">
+        <ProjectReference Include="..\..\..\..\BuildingBlocks\src\UnitTestTools\UnitTestTools.csproj" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(Configuration)' != 'Release' And !$(MSBuildProjectFile.Contains('Cleanup.csproj')) And !$(MSBuildProjectFile.Contains('SanityCheck.csproj')) And !$(MSBuildProjectFile.Contains('Devices.AdminCli.csproj'))">
```

Comment:
Considering the two conditions are the same, couldn't you merge the two ItemGroups?

Reply:
The separation was on purpose, because you usually keep project and package references in different item groups.

Reply:
be that as it may, would there be a way of keeping the two item groups but providing the condition only once?

Reply:
@daniel-almeida-konkconsulting ?

Reply:
By looking at the docs (https://learn.microsoft.com/en-us/aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/understanding-the-project-file#the-anatomy-of-a-project-file), there doesn't seem to be an aggregator we can use for the ItemGroups, so I would say we have to keep the duplicate condition.